include/glow/Graph/Graph.h

```diff
@@ -606,6 +607,45 @@ class Function final : public Named {
                                                  NodeValue indices,
                                                  NodeValue lengths);
 
+  /// Create and \returns a node of \p name, performing the SparseLengthsSum
+  /// operation, using fused rowwise quantization for the input \p data wherein
+  /// the scales and offsets are fused inline with each row of data. Gathers
```

Comment:
can you comment on how they are fused, maybe point to description in base.h

Reply:
I added a sentence saying data must have `ElemKind::Int8FusedQTy`. `ElemKind::Int8FusedQTy` isn't really specific to this node so I don't think it makes sense to explain the data layout there -- I expanded on the explanation a bit in `Quantization.md`.